### PR TITLE
🐛 Fix typo in `prompt.txt`

### PR DESCRIPTION
**Pull request by XintianS**
Fix typo in instruction No.

Change the request language of the generate command to German to generate a German dataset.

### CODE REVIEW

**Review by DavidFarago — APPROVED** [review]
Since you should leave out any unnecessary information in your prompt, and I do not see any point referencing another point, I would actually switch to an unnumbered list since the numbers are unnecessary information. But my suspicion is that this makes little to no difference.

File: prompt.txt [hunk 1]
```diff
@@ -4,11 +4,11 @@ Here are the requirements:
...
-3. The instructions should be in English.
-4. The instructions should be 1 to 2 sentences long. Either an imperative sentence or a question is permitted.
-5. You should generate an appropriate input to the instruction. The input field should contain a specific example provided for the instruction. It should involve realistic data and should not contain simple placeholders. The input should provide substantial content to make the instruction challenging but should ideally not exceed 100 words.
-6. Not all instructions require input. For example, when a instruction asks about some general information, "what is the highest peak in the world", it is not necssary to provide a specific context. In this case, we simply put "<noinput>" in the input field.
-7. The output should be an appropriate response to the instruction and the input. Make sure the output is less than 100 words.
+4. A GPT language model should be able to complete the instruction. For example, do not ask the assistant to create any visual or audio output. For another example, do not ask the assistant to wake you up at 5pm or set a reminder because it cannot perform any action.
+5. The instructions should be in German.
```
Comment (DavidFarago) [inline prompt.txt:8]: Come to think of it, wouldn't "The instructions and the corresponding inputs and outputs" be more accurate?

But we can postpone these prompt improvements if you rather want to stick as closely as possible to the original prompt...

**Comment by XintianS** [comment]
> Since you should leave out any unnecessary information in your prompt, and I do not see any point referencing another point, I would actually switch to an unnumbered list since the numbers are unnecessary information. But my suspicion is that this makes little to no difference.

We can try unnumbered list later. How about let's keep the numbers for now since Alpaca used numbered list.

File: prompt.txt (hunk 1 above)
Comment (XintianS) [reply prompt.txt:8]: I created a story to keep the suggestions in record.
https://app.shortcut.com/diwoblood/story/98588/try-out-possible-improvements-on-the-prompt-used-by-alpaca

**Pull request merged**